### PR TITLE
Highlight selected circlemarker 121

### PR DESCRIPTION
**Pull request by kevinashworth**
This is a stab at adding a color ring around the selected marker. I chose to use library called Colors to darken the current color. Feel free to ditch that and use, say, a dark grey instead. And you @emillipede  closed issued 121 already, so feel free to ignore this PR completely!

### CODE REVIEW

File: public/js/Map.js [hunk 1]
```diff
@@ -124,11 +124,17 @@ var app = this.app || {};
...
     if (action === 'shrink') {
       marker.setStyle({
-        radius: currentRadius / 2
+        radius: currentRadius / 1.75,
+        stroke: false
       });
     } else {
+      var HSVColor = $c.hex2hsv(marker.options.fillColor);
+      var RGBColor = $c.hsv2rgb(HSVColor.H, HSVColor.S, HSVColor.V * 0.75);
+      var newColor = $c.rgb2hex(RGBColor.R, RGBColor.G, RGBColor. B);
```
Comment (emillipede) [inline public/js/Map.js]: instead of newColor, what about ringColor?

**Comment by emillipede** [comment]
Thanks @kevinashworth, I like this approach! 

I'm noticing a small issue with how the rings around the marker work right now. If a user changes the palette/view (eg. from native to common name), the marker's color ring isn't automatically updating. The ring color stays consistent with whichever color the marker was in the earlier view (I've included an example in the screenshot). If I zoom in or out in the new view, the color ring does update to match the new color palette.

@ian-r-rose I think you came across a similar problem with the old species drop down. How did you fix it?

Screen Shot 2019-07-11 at 8 48 55 PM

**Comment by kevinashworth** [comment]
Hey @emillipede I pushed a potential fix for case where the palette is changed.

**Comment by emillipede** [comment]
cool, thanks @kevinashworth! this update looks good to me

the only change I suggest is updating the name of the variable to ringColor, newColor is a little confusing

**Comment by emillipede** [comment]
thank you @kevinashworth !

**Pull request merged**